tests/time_tests/scripts/run_timetest.py

```diff
@@ -55,6 +55,22 @@ def run_cmd(args: list, log=None, verbose=True):
     return proc.returncode, ''.join(output)
 
 
+def parse_stats(stats: list, res: dict):
```

Comment:
I do think we need tests. Would it be a problem to add test_run_timetest.py with few tests for parse_stats?

Reply:
I don't really understand what the test will have to check? multiple pre-prepared yml files?

Reply:
add one test for parse_stats in run_timetest.py